packages/@angular/cli/commands/eject.ts

```diff
@@ -0,0 +1,22 @@
+import { tags } from '@angular-devkit/core';
+import { Command, Option } from '../models/command';
+
+
+export default class HelpCommand extends Command {
```

Comment:
`HelpCommand` -> `EjectCommand`, also a couple of other places below.

Reply:
Done.